weblab_cg/data/tests/chaste_reference_models/Normal/additional_models/Shannon2004.cpp

```diff
@@ -1,6 +1,6 @@
 //! @file
 //! 
-//! This source file was generated from CellML.

```

Comment:
Not sure how easy it would be to exclude these kinds of change from commits? They make things very noisy!

Reply:
Indeed, I'll look into it.